### PR TITLE
Searching Sorting

### PR DESCRIPTION
**Pull request by mieraldudhagara02**
# Sorting and Searching

### Sorting
you can select any category among sneakers, wallet and cap. You will see only that selected category's products.

### Searching
After sorting, if you want any particular product of that selected category, you can also search it into input field.

### CODE REVIEW

**Review by utsavsomaiya — CHANGES_REQUESTED** [review]
image

Is it needed to push?

**Comment by utsavsomaiya** [comment]
image

File: products.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+//     increment.classList.add('increment')
+
+//     const ordered_product_price = document.createElement('span');
+//     ordered_product_price.classList.add('ordered-product-price')
+//     ordered_product_price.innerHTML = product.price;
+
+//     orderDiv.appendChild(Ordered_product_image)
+//     orderDiv.appendChild(orderedProductName)
+//     orderDiv.appendChild(decrement)
+//     orderDiv.appendChild(quantity)
```
Comment (utsavsomaiya) [inline products.js:192]: I dont know what is this commented code?

**Review by utsavsomaiya — CHANGES_REQUESTED** [review]

File: products.js (hunk 1 above)
Comment (mieraldudhagara02) [reply products.js:192]: wip

Comment (utsavsomaiya) [reply products.js:192]: You give the title searching and sorting then why this code in this?

Comment (mieraldudhagara02) [reply products.js:192]: i'll make it correct

Comment (utsavsomaiya) [reply products.js:192]: Not needed!

**Pull request closed without merging**